packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -139,7 +139,7 @@ library TaikoData {
         // Slot 7: never or rarely changed
         uint64 genesisHeight;
         uint64 genesisTimestamp;
-        uint64 __reserved71;
+        uint64 proofTimeTarget;
```

Comment:
It seems using `__reserved91` for `proofTimeTarget` will save some gas as neither `genesisHeight` or `genesisTimestamp` is accessed when block is proposed/proven.